test/unit/ajax.js

```diff
@@ -1197,6 +1197,25 @@ module( "ajax", {
 		{
 			url: url("data/json.php"),
 			converters: {
+				"text json": function( data ) {
+					ok( true, "promise converter called" );
+					return jQuery.Deferred().resolveWith( data );
```

Comment:
This is not sufficient coverage, because `jQuery.Deferred().resolveWith( data )` is synchronous.